docs/content/docs/dev/table/sql/queries/hints.md

```diff
@@ -84,4 +84,219 @@ insert into kafka_table1 /*+ OPTIONS('sink.partitioner'='round-robin') */ select
 
 ```
 
+## Join Hints
+
+{{< label Batch >}}
+
+Join hints allow users to suggest the join strategy to optimizer in order to get a more optimal execution plan. 
+Currently, only supported in batch mode.
+
+### Supported Join Hints
+The following Join strategies are currently supported in Flink SQL for batch mode:
```

Comment:
We should focus on join hint instead of join strategy

Reply:
> We should focus on join hint instead of join strategy

Already deleted the desc about join strategies.